android/titanium/build.gradle

```
@@ -82,50 +87,45 @@ android {
 		main {
 			manifest.srcFile 'AndroidManifest.xml'
 			assets.srcDirs = [
-				'assets',
-				"${projectDir}/../modules/ui/assets"
+					'assets',
+					"${projectDir}/../modules/ui/assets"
 			]
 			java.srcDirs = [
-				'src/java',
-				"${projectDir}/../modules/accelerometer/src/java",
-				"${projectDir}/../modules/analytics/src/java",
-				"${projectDir}/../modules/android/src/java",
-				"${projectDir}/../modules/app/src/java",
-				"${projectDir}/../modules/calendar/src/java",
-				"${projectDir}/../modules/contacts/src/java",
-				"${projectDir}/../modules/database/src/java",
-				"${projectDir}/../modules/filesystem/src/java",
-				"${projectDir}/../modules/geolocation/src/java",
-				"${projectDir}/../modules/gesture/src/java",
-				"${projectDir}/../modules/locale/src/java",
-				"${projectDir}/../modules/media/src/java",
-				"${projectDir}/../modules/network/src/java",
-				"${projectDir}/../modules/platform/src/java",
-				"${projectDir}/../modules/ui/src/java",
-				"${projectDir}/../modules/utils/src/java",
-				"${projectDir}/../modules/xml/src/java",
-				"${projectDir}/../runtime/common/src/java",
-				"${projectDir}/../runtime/v8/src/java"
-			]
-			jni.srcDirs = [
-				"${projectDir}/../runtime/v8/src/native",
-				"${projectDir}/../runtime/v8/src/native/modules",
-				"${projectDir}/../runtime/v8/generated"
+					'src/java',
+					"${projectDir}/../modules/accelerometer/src/java",
+					"${projectDir}/../modules/analytics/src/java",
+					"${projectDir}/../modules/android/src/java",
+					"${projectDir}/../modules/app/src/java",
+					"${projectDir}/../modules/calendar/src/java",
+					"${projectDir}/../modules/contacts/src/java",
+					"${projectDir}/../modules/database/src/java",
+					"${projectDir}/../modules/filesystem/src/java",
+					"${projectDir}/../modules/geolocation/src/java",
+					"${projectDir}/../modules/gesture/src/java",
+					"${projectDir}/../modules/locale/src/java",
+					"${projectDir}/../modules/media/src/java",
+					"${projectDir}/../modules/network/src/java",
+					"${projectDir}/../modules/platform/src/java",
+					"${projectDir}/../modules/ui/src/java",
+					"${projectDir}/../modules/utils/src/java",
+					"${projectDir}/../modules/xml/src/java",
+					"${projectDir}/../runtime/common/src/java",
+					"${projectDir}/../runtime/v8/src/java"
 			]
 			res.srcDirs = [
-				'res',
-				"${projectDir}/../modules/ui/res"
+					'res',
+					"${projectDir}/../modules/ui/res"
```

Intentional indention?

```suggestion
				'res',
				"${projectDir}/../modules/ui/res"
```